README.rst

```diff
@@ -81,13 +146,95 @@ Creating a Cluster
 Running a Relay
 ---------------
 
+Now, with the Replication Set all set up, you can run a relay process.
+
 ::
 
     pgshovel-relay configuration.yml example
 
+
+The relay takes a yaml configuration file, which is in the following format.
+
+::
+
+    stream:
+        path: module.path.to:WriterObject
+        configuration:
+            key: value
+            key2: value2
+
+
+The Relay will attempt to load the Writer object defined at path, calling ``.configure(configuration)`` with the ``configuration`` being the keys and values defined via the ``configuration`` key.  This ``configure`` method is responsible for returning the newly constructed writer instance.
+
+For instance, to use the built in ``KafkaWriter`` at the hostname ``kafka``, use the following config file.
+
+::
+    stream:
+        path: pgshovel.relay.streams.kafka:KafkaWriter
+        configuration:
+            hosts: kafka
+
+
+Once started, the Relay worker runs as a daemon, relaying raw database mutations to the output stream for usage. The Writer instance just needs to respond to the ``.push(messages)`` API, where ``messages`` is a sequence of ``pgshovel.interfaces.streams.MutationOperation`` objects.
+
+
+Replication Worker
+------------------
+
+A Replication Worker is used to replicate a stream of Batch Operations into a new database replication target.  It is started with the ``pgshovel-replicate`` command, passing the path to the replication config file as well as the replicaion set name.
+
+::
+
+    python-replicate example/configurations/replication.yml <replication-set>
+
+The config file looks very similar to the relay config file, though it contains three sections: ``loader``, ``stream``, and ``target``, corresponding to the three components of Replication as described in the "Concepts" section above.
+
+::
+
+    loader:
+        path: module.path.to:Loader
+
+    stream:
+        path: module.path.to:Stream
+        configuration:
+            key: value
+
+    target:
+        path: module.path.to:Target
+        configuration:
+            key: value
+            key2: value2
+
+For example, here is a configuration file which loads data using the simple loder, streams in further mutations via the kafka stream and replicates those changes to the Postgres target
```

Comment:
- "loder" should be "loader",
- "kafka" should be "Kafka",
- no terminal punctuation